scripts/r2s.py

```diff
@@ -140,10 +140,10 @@ def step1():
     ves = list(mesh.iter_ve())
     tags_keep = ("cell_number", "cell_fracs",
                  "cell_largest_frac_number", "cell_largest_frac")
-    for tag in mesh.mesh.getAllTags(ves[0]):
+    for tag in mesh.get_all_tags():
         if tag.name not in tags_keep:
-            mesh.mesh.destroyTag(tag, True)
-    mesh.mesh.save('blank_mesh.h5m')
+            mesh.delete(tag)
```

Comment:
Error jumps out at this line:
```
Traceback (most recent call last):
  File "/root/.local/bin/r2s.py", line 223, in <module>
    main()
  File "/root/.local/bin/r2s.py", line 218, in main
    step1()
  File "/root/.local/bin/r2s.py", line 145, in step1
    mesh.delete(tag)
AttributeError: 'MeshTally' object has no attribute 'delete'
```

Reply:
Sorry, the delete tag function hasn't been merged. So, it of course has no attribute 'delete'.